omod/src/main/webapp/resources/htmlforms/triage.html

```diff
@@ -54,6 +54,18 @@
                 jq('#recent-calculated-bmi').html("");
             }
         }
+        function zScoreSelected() {
+            var zScore = getField('z-score.value').val();
+            if (zScore == 1115) {
```

Comment:
Hi @patryllus, thanks for your work on this. I would appreciate some clarification regarding the values 1115 and 123814. Specifically, I'm interested in understanding their significance within the context of this code. Furthermore, could you elaborate on the reasoning behind these numbers being mapped to 0, -1, -2, and -3? This additional information will be instrumental in ensuring a comprehensive understanding of your proposed changes. Thanks in advance!

Reply:
![z-score](https://github.com/palladiumkenya/openmrs-module-kenyaemr/assets/1901372/025f3e63-18b4-43fe-9752-f5890



@donaldkibet  we gave z-score question on triage form for children under 5 years with concept   <td> <br/> Z-Score </td>
                        <td><br/>
                            <obs conceptId="163515AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
                                 answerConceptIds="1115AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
                                                   123814AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
                                                   123815AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
                                                   164131AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
                                 answerLabels="Normal (Median),Mild (-1 SD),Moderate (-2 SD),Severe (-3 SD and -4 SD)"
                                 style="dropdown"
                                 id="z-score"/>
                        </td>   

The above answers values 1115 and 123814. This are outcomes/interpretations of the Z-score and are value coded. The actual score are Numeric and are implicitly saved are 0,-1,-2,-3 for Normal,Mild,Moderate and Severe. Hope this answers your questions.

Reply:
<td> <br/> Z-Score </td>
                        <td><br/>
                            <obs conceptId="163515AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
                                 answerConceptIds="1115AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
                                                   123814AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
                                                   123815AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,
                                                   164131AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
                                 answerLabels="Normal (Median),Mild (-1 SD),Moderate (-2 SD),Severe (-3 SD and -4 SD)"
                                 style="dropdown"
                                 id="z-score"/>
                        </td>